en/appendices/new-features-in-cakephp-1-3.rst

```diff
@@ -181,8 +181,7 @@ could do::
 
 -  Missing Validation methods now trigger errors, making debugging
    why validation isn't working easier.
--  Models now support
-   `virtual fields <http://book.cakephp.org/view/1608/Virtual-fields>`_
+-  Models now support virtual fields.
```

Comment:
Should this link to http://book.cakephp.org/3.0/en/orm/entities.html#creating-virtual-fields